website/pages/how-to-guides/how-to-load-infrastructure-data-into-athena.mdx

```diff
@@ -244,6 +244,10 @@ WHERE "block_public_acls" <> TRUE
 
 Wait, isn't that the bucket we just created for this guide? Better go lock it down 😉
 
+<Callout type="info">
+  Athena is case-insensitive. If you see an error like `Row is not a valid JSON Object - JSONException: Duplicate key`, this means some of the data in a JSON column contained keys where only the case differed (e.g. `Name` and `name`). AWS tags are case sensitive, so this can be a common cause. For more information about ways to address this issue, see [this Github comment](https://github.com/cloudquery/cloudquery/issues/7639#issuecomment-1419082572).
```

Comment:
```suggestion
  Athena is case-insensitive. If you see an error like `Row is not a valid JSON Object - JSONException: Duplicate key`, this means some of the data in a JSON column contained keys where only the case differed (e.g. `Name` and `name`). AWS tags are case sensitive, so this can be a common cause. For more information about ways to address this issue, see [this GitHub comment](https://github.com/cloudquery/cloudquery/issues/7639#issuecomment-1419082572).
```